packages/datadog-plugin-aws-sdk/src/services/bedrockruntime.js

```diff
@@ -0,0 +1,236 @@
+  const modelId = params.modelId
+
+  if (provider === AI21) {
+    const temperature = requestBody.temperature || ''
+    const topP = requestBody.topP || ''
+    const maxTokens = requestBody.maxTokens || ''
+    const stopSequences = requestBody.stopSequences || []
+    let prompt = requestBody.prompt
+
+    if (modelId.includes('jamba')) {
+      for (const message of requestBody.messages) {
+        if (message.role === 'user') {
+          prompt = message.content // Return the content of the first user message
+        }
+      }
+    }
+    return {
+      prompt,
+      temperature,
+      top_p: topP,
+      max_tokens: maxTokens,
+      stop_sequences: stopSequences
+    }
+  } else if (provider === AMAZON && modelId.includes('embed')) {
+    return { prompt: requestBody.inputText }
+  } else if (provider === AMAZON) {
+    const textGenerationConfig = requestBody.textGenerationConfig || {}
+    return {
+      prompt: requestBody.inputText,
+      temperature: textGenerationConfig.temperature || '',
+      top_p: textGenerationConfig.topP || '',
+      max_tokens: textGenerationConfig.maxTokenCount || '',
+      stop_sequences: textGenerationConfig.stopSequences || []
+    }
+  } else if (provider === ANTHROPIC) {
+    const prompt = requestBody.prompt || ''
+    const messages = requestBody.messages || ''
+    return {
+      prompt: prompt || messages,
+      temperature: requestBody.temperature || '',
+      top_p: requestBody.top_p || '',
+      top_k: requestBody.top_k || '',
+      max_tokens: requestBody.max_tokens_to_sample || '',
+      stop_sequences: requestBody.stop_sequences || []
+    }
+  } else if (provider === COHERE && modelId.includes('embed')) {
+    return {
+      prompt: requestBody.texts,
+      input_type: requestBody.input_type || '',
+      truncate: requestBody.truncate || ''
+    }
+  } else if (provider === COHERE) {
+    return {
+      prompt: requestBody.prompt,
+      temperature: requestBody.temperature || '',
+      top_p: requestBody.p || '',
+      top_k: requestBody.k || '',
+      max_tokens: requestBody.max_tokens || '',
+      stop_sequences: requestBody.stop_sequences || [],
+      stream: requestBody.stream || '',
+      n: requestBody.num_generations || ''
+    }
+  } else if (provider === META) {
+    return {
+      prompt: requestBody.prompt,
+      temperature: requestBody.temperature || '',
+      top_p: requestBody.top_p || '',
+      max_tokens: requestBody.max_gen_len || ''
+    }
+  } else if (provider === MISTRAL) {
+    return {
+      prompt: requestBody.prompt,
+      max_tokens: requestBody.max_tokens || '',
+      stop_sequences: requestBody.stop || [],
+      temperature: requestBody.temperature || '',
+      top_p: requestBody.top_p || '',
+      top_k: requestBody.top_k || ''
+    }
+  } else if (provider === STABILITY) {
+    // TODO: request/response formats are different for image-based models. Defer for now
+    return {}
+  }
```

Comment:
I know we have this written out explicitly in the Python integration, but we can probably just leave this branch off. It'll hit the final `return {}` anyways.